rules/macos/defense_evasion_sandboxed_office_app_suspicious_zip_file.toml

```diff
@@ -0,0 +1,32 @@
+[metadata]
+creation_date = "2020/01/11"
+maturity = "production"
+updated_date = "2020/01/11"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of suspicious zip file prepended with special chars. Sandbox Microsoft Office applications are
+allowed to write files that start with special characters which can be combined with autostart location to achieve
+sandbox evasion.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Potential Office Sandbox Evasion"
+references = [
+    "https://i.blackhat.com/USA-20/Wednesday/us-20-Wardle-Office-Drama-On-macOS.pdf",
+    "https://www.mdsec.co.uk/2018/08/escaping-the-sandbox-microsoft-office-on-macos/",
+    "https://desi-jarvis.medium.com/office365-macos-sandbox-escape-fcce4fa4123c",
+]
+risk_score = 73
+rule_id = "d22a85c6-d2ad-4cc4-bf7b-54787473669a"
+severity = "high"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Defense Evasion"]
+type = "query"
+
+query = '''
+event.category:file and not event.type:deletion and file.name:~$*.zip
+'''
+
```

Comment:
We could add the Mitre technique `Virtualization/Sandbox Evasion` https://attack.mitre.org/techniques/T1497/. That's likely one we don't have covered, so would be a good addition.

Reply:
indeed but their description for T1497 is not covering process sandbox/AppContainer evasion.